tests/perf-test/OptionsListUtils.perf-test.ts

```diff
@@ -105,9 +105,12 @@ describe('OptionsListUtils', () => {
     });
 
     /* Testing getFilteredOptions */
-    test('[OptionsListUtils] getFilteredOptions with search value', async () => {
+    test('[OptionsListUtils] getFilteredOptions', async () => {
         await waitForBatchedUpdates();
-        await measureFunction(() => OptionsListUtils.getFilteredOptions(options.reports, options.personalDetails, mockedBetas, SEARCH_VALUE));
+        // Ideally getFilteredOptions should not be used with both options and search value
+        // The more performant filterOptions should be used instead to pass search value with options containing reports and personal details
```

Comment:
I think we need to rephrase this comment. 
```suggestion
        // It's recommended not to use getFilteredOptions with both options and a search value
        // For better performance, use filterOptions instead, especially when passing a search value with options that include reports and personal details.
```